content/en/docs/release-notes/release-notes-0.14.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Release Notes"
+linkTitle: "v0.14"
+weight: 140
+type: "docs"
+---
+
+In `v0.14` we've focused on a number of areas:
+
+## Webhook changes
+
+The webhook component has now been made required, meaning any users that previously used the `no-webhook` variant will
+need to upgrade their installations to enable the webhook. This will be done automatically upon upgrading to the `v0.14`
+release, so no additional action is required.
+
+If you have issues running the webhook in your environment, we'd like to hear from you! We are aware of issues relating
+to firewall rules from the Kubernetes API server to the webhook pod(s) - we would like to gather together a corpus of
+configuration snippets that can be used to ensure the webhook is successfully deployed in these environments too.
+
+This change is required in order to support the upcoming changes to our API versions, as we introduce `v1alpha3`,
+`v1beta1` and `v1` over the coming months!
+
+## Improving our deployment and release process
+
+After reports of various issues installing on older Kubernetes and OpenShift versions, we've taken some time to revise
+our installation manifests.
+
+There are now two 'variants' to choose from, 'standard' and the 'legacy', with a simple way to know which to use:
+
+| Environment          | Variant to use             |
+| -------------------- | -------------------------- |
+| Kubernetes 1.15+     | `cert-manager.yaml`        |
+| OpenShift 4          | `cert-manager.yaml`        |
+| Kubernetes 1.11-1.14 | `cert-manager-legacy.yaml` |
+| OpenShift 3.11       | `cert-manager-legacy.yaml` |
+
+Please be sure to read the upgrade guide for more information on how to upgrade from a previous release.
+
+## `CustomResourceDefinition` conversion webhook + `v1alpha3` API version
+
+As part of the effort to mature our API, we are releasing the `v1alpha3` API version. This contains a number of small
+changes, notably moving some fields to the `subject` stanza on the Certificate resource to be more consistent with how
+certain options are specified.
+
+With this we have enabled the 'conversion webhook', which enables API clients to utilize both the `v1alpha2` and
+`v1alpha3` APIs simultaneously, similar to other core resources in Kubernetes.
+
+Thanks to this conversion webhook, this upgrade and future upgrades after it should be seamless. The ability to make
+these kinds of changes to our API will enable the `v1beta1` API version to be released in a seamless manner in an
+upcoming release too.
+
+## Support for Kubernetes 1.11 and OpenShift 3.11
+
+We've had a number of users who are using OpenShift 3.11 & Kubernetes 1.11 reach out requesting support with
+installation. In this release, we've expanded the range of Kubernetes versions we support to once again include 1.11,
+as well as adding support for OpenShift 3.11.
```

Comment:
```suggestion
By popular demand, we've expanded our range of tested and supported platforms to include Kubernetes 1.11 and OpenShift 3.11.
```

Reply:
Maybe also add a link to our supported platforms documentation, and / or to the testing grid page.

Reply:
I'd rather not link to/expose testgrid here - I'd rather us make notes and clearly communicate to users whether something is supported or not. It could cause confusion to users if we have issues in future with test-infra, leading them to think it's not compatible. I'd prefer we link to testgrid or something from a new/updated 'contributing' section :)

We also don't have an explicit 'list of supported platforms' aside from advising that we work on OpenShift 3.11 and Kubernetes 1.11+ - this would be something good to add (but right now, we need to get these release notes out 🙈)